Aquarius/Src/Aquarius/Renderer/Shader.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <iostream>
+#include <glm/glm.hpp>
+#include <unordered_map>
+
+namespace Aquarius {
+
+    class Shader
+    {
+    public:
+        // Path to vertex and fragment shader source code
+        Shader(const std::string& vertexPath, const std::string& fragmentPath);
+        ~Shader();
```

Comment:
Should we maybe delete the default constructor? I think the compiler will generate one where the only data member is a uint32

Reply:
I tested this, it doesnt create a constructor with uint32 parameter or even a default one with no params (super weird). I think this can for sure happen sometimes though, according to the vague docs ("Sometimes a constructor is implicitly declared")

I will delete the default constructor tho, probably good practice